vtex/utils/transform.ts

```diff
@@ -1175,6 +1192,7 @@ export function toPlace(
     latitude,
     longitude,
     name,
+    pickupHolidays,
```

Comment:
Use `specialOpeningHoursSpecification` instead of `pickupHolidays`
```suggestion
    specialOpeningHoursSpecification: pickupHolidays,
```